go/pkg/client/cas.go

```diff
@@ -290,12 +290,9 @@ func (c *Client) BatchDownloadBlobs(ctx context.Context, dgs []digest.Digest) (m
 func (c *Client) makeBatches(dgs []digest.Digest) [][]digest.Digest {
 	var batches [][]digest.Digest
 	log.V(2).Infof("Batching %d digests", len(dgs))
-	sort.Slice(dgs, func(i, j int) bool {
```

Comment:
Doubt: Why remove this?

Reply:
If we keep this, file writes for ouputs happen in almost random order. But this PR wants file writes happen in nearly sorted by path to have locality.

Reply:
Ack.